src/js/claims-status/actions/index.jsx

```diff
@@ -94,10 +94,23 @@ export function getAppeals(filter) {
 }
 
 export function fetchAppealsSuccess(response) {
+  const appeals = response.data;
+  const v1ToV2IdMap = {};
+  appeals.forEach(appeal => {
+    // In case there are no v1 appeal ids
+    if (!appeal.attributes.appealIds) {
+      return;
+    }
+
+    appeal.attributes.appealIds.forEach(id => {
+      v1ToV2IdMap[id] = appeal.id;
```

Comment:
What is the plan to use this? Would it make sense to create an object with the v2 appeal id as the key, and the appeals as the values?